datameta/api/groups.py

```diff
@@ -110,6 +117,32 @@ class ChangeGroupName(DataHolderBase):
     new_group_name: str
 
 
+@view_config(
+    route_name="groups_id",
+    renderer="json",
+    request_method="GET",
+    openapi=True
+)
+def get(request: Request):
+
+    # Authenticate the user
+    auth_user = security.revalidate_user(request)
+
+    group_id = request.matchdict["id"]
+    db = request.dbsession
+
+    # Get the targeted user
+    target_group = resource_by_id(db, Group, group_id)
```

Comment:
The return value has to be checked for `None`. Requesting an invalid group ID currently leads to an uncaught exception.

Reply:
i'm raising 404, do you mean 403?

Reply:
Yes, sorry, 403. Please change.